tools/manticore-analysis/analysis.md

```diff
@@ -0,0 +1,56 @@
+# Manticore
+[Manticore](https://github.com/trailofbits/manticore) is a versatile symbolic execution tool that can analyze Ethereum Smart Contracts,
+Linux binaries, and Windows binaries. It's widely used for security auditing and testing of
+applications where security is paramount. Its main features include:
+- **Program Exploration**: executing a program with symbolic inputs and exploring all the possible states it can reach,
+- **Input Generation**: producing concrete inputs that result in a given program state,
+- **Error Discovery**: detecting crashes and other failure cases in binaries and Smart Contracts,
+- **Instrumentation**: providing fine-grained control of state exploration via event callbacks and instruction hooks,
+- **Programmatic Interface**: exposing programmatic access to its analysis engine via a Python API.
+
+Manticore can analyze the following types of programs:
+
+- Ethereum Smart Contracts (EVM bytecode)
+- Linux ELF binaries (x86, x86_64, aarch64, and ARMv7)
+- WASM Modules
+
+## Installation and execution:
+- Installing by PIP results in [protobuf incorrect version error](sample_execution/execution_errors.md#pip-instalation)
+  (described better here: https://github.com/trailofbits/manticore/issues/2600)
+- Build attempt with docker image version 3.7+ result in [attribute error](sample_execution/execution_errors.md#docker-v37-and-latest-)
+- Build attempt with docker image version 3.6 results in [tool custom exception](sample_execution/execution_errors.md#docker-v36)
+- Manticore may be built on docker image version 3.5. Execution example may be found [here](sample_execution/sample_execution.md),
+  and output files generated by the Manticore may be found under: `./Manticore/sample_execution_outputs`
+> **Support**:
+> This project is no longer internally developed and maintained. The team responsible for creating this tool announced its
+> readiness to review and accept small, well-written pull requests by the community (only bug fixes and minor
+> enhancements shall be considered). But there have veen no changes in the tool`s codebase since December 2022. All error
+> executions may be found in [this file](sample_execution/execution_errors.md)
+
+### Custom detector investigation
+Manticore has no documented ways to introduce new detectors. It requires adding a new detector class to [detectors.py](https://github.com/trailofbits/manticore/blob/master/manticore/ethereum/detectors.py) file and importing it in cli.py (for command line interface analysis).
```

Comment:
Do we need to add custom detectors? If so, how can we add them if installation by PIP results in error?